api/entity/v1/reqres/secret.go

```diff
@@ -5,17 +5,19 @@
 package reqres
 
 import (
-	"time"
-
 	"github.com/spiffe/spike-sdk-go/api/entity/data"
 )
 
-// SecretResponseMetadata is meta information about secrets for internal
-// tracking.
-type SecretResponseMetadata struct {
-	CreatedTime time.Time  `json:"created_time"`
-	Version     int        `json:"version"`
-	DeletedTime *time.Time `json:"deleted_time,omitempty"`
+// SecretMetadataRequest for get secrets metadata
+type SecretMetadataRequest struct {
+	Path    string `json:"path"`
+	Version int    `json:"version,omitempty"` // Optional specific version
```

Comment:
Some of these `version`s were `string` instead of `int`; updated them.

Not sure how it will impact the `spike` repo; will fix it it results in build failures etc.